networkit/generators.pyx

```diff
@@ -743,23 +1008,56 @@ cdef class LFRGenerator(Algorithm):
 
 	def getPartition(self):
 		"""
+		getPartition()
+
 		Return the generated Partiton.
 
-		Returns:
-		--------
+		Returns
+		-------
 		networkit.Partition
 			The generated partition.
 		"""
 		return Partition().setThis((<_LFRGenerator*>(self._this)).getPartition())
 
 	@classmethod
 	def setPathToReferenceImplementationDir(cls, path):
+		"""
+		setPaths(path)
+
+		Helper function for fit. Sets an output folder for writing the result to.
+
+		Parameters
+		----------
+		path : str
+			String containing a path.
+		"""
 		cls.paths["refImplDir"] = path
 
 
 	@classmethod
 	def fit(cls, Graph G, scale=1, vanilla=False, communityDetectionAlgorithm=PLM, plfit=False):
```

Comment:
It's a bit weird to have `cls` instead of `self`. Should we change it? Maybe on a separate PR.

Reply:
I also wondered - maybe indeed split that into another PR.